command-parser.js

```diff
@@ -330,8 +330,8 @@ class CommandContext {
 				return false;
 			}
 
-			if (!this.room.banwordRegex) this.updateBanwords();
-			if (this.room.banwordRegex !== true && this.room.banwordRegex.test(message) && !user.can('mute', null, this.room)) {
+			if (!room.banwordRegex) this.updateBanwords();
```

Comment:
Staying as `this.room` also wouldn't fix it. The way it's written currently, you'd need to wrap the entire thing in an `if (room)` check.